lib/fourOhFour.js

```diff
@@ -64,7 +66,7 @@ function fourOhFour (options) {
   function createOnBadUrl () {
     return function onBadUrl (path, req, res) {
       const id = genReqId(req)
-      const childLogger = logger.child({ reqId: id })
+      const childLogger = createChildLogger(this, logger, req, id, options)
```

Comment:
For this scenario, given the nature of the `forOhFour`, let's fallback inoto the `kFourOhFourLevelInstance` which directly points to the instance owner of the 404 handler

Reply:
You're right, missed that 👍